actors/builtin/miner/testing.go

```diff
@@ -56,6 +79,9 @@ func CheckStateInvariants(st *State, store adt.Store) (*StateSummary, *builtin.M
 	faultyPower := NewPowerPairZero()
 
 	// Check deadlines
+	acc.Require(st.CurrentDeadline < WPoStPeriodDeadlines,
+		"current deadline index is greater than deadlines per period(%d): %d", WPoStPeriodDeadlines, st.CurrentDeadline)
```

Comment:
As follow-up, I think we should also accept epoch as a parameter and make dependent state checks like:
- proving period start
- expiration queues have been processed
- no faults are too old
etc